### PR TITLE
Add scripts to run docker and build jdk

### PR DESCRIPTION
**Pull request by vasyoid**
Here are some scripts that automate some frequent actions. They all must be executed in the root repo directory.

These run the corresponding docker images:
```
dev-riscv/scripts/run_toolchain.sh
dev-riscv/scripts/run_qemu.sh
```

These configure and run build. They have the following parameters:
* **-h** show help
* **-v** set --with-jvm-variants flag
* **-l** set --with-debug-level flag
 ```
dev-riscv/scripts/configure.sh
dev-riscv/scripts/make.sh
```

Any critique and improvements are welcome :)

### CODE REVIEW

File: dev-riscv/scripts/configure.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        ;;
+    l)  level=$OPTARG
+        ;;
+    esac
+done
+
+bash configure \
+    --with-jvm-variants=$variants \
+    --with-debug-level=$level \
+    --with-native-debug-symbols=internal \
```
Comment (vasyoid) [inline dev-riscv/scripts/configure.sh:29]: The debug flags are taken from here: https://hg.openjdk.java.net/jdk-updates/jdk9u/raw-file/tip/common/doc/building.html#common-configure-arguments

With these flags gdb server **does not** show the following error:
```
No debugging symbols found in build/linux-riscv64-core-fastdebug/jdk/bin/java
```

However, the client shows the following:
```
No symbol table is loaded.  Use the "file" command.
```

**Review by AntonKozlov — APPROVED** [review]

**Comment by tsarn** [comment]
I think it could be a good idea to make these scripts independent of current working directory.
Also, as far as I know, `./riscv-sysroot` is not documented anywhere as a standard path for sysroot. For example, on my system it lives in `/opt/riscv/sysroot`. Maybe it could be passed in through an environment variable or a parameter?

**Comment by vasyoid** [comment]
> I think it could be a good idea to make these scripts independent of current working directory.

What do you mean by independence of current working directory? Should the path to `jdk-riscv` directory be passed in?

> Also, as far as I know, `./riscv-sysroot` is not documented anywhere as a standard path for sysroot. For example, on my system it lives in `/opt/riscv/sysroot`. Maybe it could be passed in through an environment variable or a parameter?

If sysroot lives outside the `jdk-riscv` folder then it has to be mounted separately. How about mounting it to another path, like `/opt/riscv/sysroot` in docker?

**Comment by tsarn** [comment]
> What do you mean by independence of current working directory?

For example, configure only works if we run it as `./dev-riscv/scripts/configure.sh` from repository root, but not as `./scripts/configure.sh` from `dev-riscv` (as it uses `bash configure` and `dev-riscv` doesn't contain a configure script). I don't know how critical this is, so maybe it's ok.

> How about mounting it to another path, like `/opt/riscv/sysroot` in docker?

Yeah, it could be done. What I meant was that I personally don't have `riscv-sysroot` directory, so maybe it should be a parameter of the script and not hardcoded into it.

**Comment by vasyoid** [comment]
Fixed scripts. Now they all can be run from any folder.
`make.sh` and `configure.sh` now just do `cd /jdk-riscv` so no parameters are needed.
`run_toolchain.sh` now takes one optional parameter -- path to `jdk-riscv` directory. The default value is `$(pwd)`.
`run_qemu.sh` now takes two optional parameters -- path to sysroot and path to `jdk-riscv` directory. The default values are `$(pwd)/riscv-sysroot` and `$(pwd)` respectively.

**Review by tsarn — APPROVED** [review]

**Comment by AntonKozlov** [comment]
@vasyoid and you can merge this too, please

**Pull request merged**